tests/robottelo/test_decorators.py

```diff
@@ -106,7 +106,8 @@ class MockBug(object):  # pylint:disable=R0903
             whiteboard = None
         dec_settings.upstream = False
         decorators._get_bugzilla_bug = lambda bug_id: MockBug()
-        for MockBug.status in BZ_OPEN_STATUSES + BZ_CLOSED_STATUSES:
+        # skip 'CLOSED' state, as it overides whiteboard 'verified in upstream'
```

Comment:
The test should include that case then, skipping is leaving an untested part of the function.

Adjust the cases to match all the expected permutations. Include when the bug is closed and has the whiteboard and when there's no whiteboard associated.